create-mango/source/app.tsx

```diff
@@ -72,15 +72,17 @@ function Wizard() {
 		return (
 			<>
 				<Text>[💽] Enter name of this project</Text>
-				<Text color={'gray'}>
-					App will be created in a folder with this name in this directory.
-				</Text>
-				<TextInput
-					onChange={(value) => setProjectName(value)}
-					value={projectName}
-					onSubmit={handleDestinationSelect}
-					showCursor
-				/>
+				<Box>
+					<Text color={'gray'}>
+						App will be created in a folder with this name in this directory.{' '}
+					</Text>
+					<TextInput
+						onChange={(value) => setProjectName(value)}
+						value={projectName}
+						onSubmit={handleDestinationSelect}
+						showCursor
+					/>
+				</Box>
```

Comment:
"App will be created ...." is long so that entering something at the end of can cause wrapped text rendering on smaller terminals. I use a small size terminal.